clients/src/main/java/org/apache/kafka/clients/consumer/internals/StreamsGroupHeartbeatRequestManager.java

```diff
@@ -126,7 +113,7 @@ public NetworkClientDelegate.PollResult poll(long currentTimeMs) {
                 "messages. You can address this either by increasing max.poll.interval.ms or by " +
                 "reducing the maximum size of batches returned in poll() with max.poll.records.");
 
-            membershipManager.transitionToSendingLeaveGroup(true);
+            membershipManager.onPollTimerExpired();
```

Comment:
why these renamings? I think it could be beneficial to not rename things too much, unless we change the behavior

Reply:
I was not satisfied with the existing names since they were not descriptive enough. In this specific case, it also allowed to make `transitionToSendingLeaveGroup()` private and to not expose the boolean parameter that made the code a bit harder to read.
Why is it beneficial to not rename the things too much?
I think we should rename if it makes the code clearer.